src/db.ts

```diff
@@ -77,29 +84,27 @@ export class NostroidDexie extends Dexie {
       })
     })
   }
-  
-  public async nip05Valid(name, pubkey) {
-    if (name == undefined || name.length < 3) {
+  private async nip05Valid(name: string, pubkey: string) {
+    if (typeof name !== 'string' || name.length < 3) {
       return false
     }
-    var n = name
+    let n = name
     switch (name.indexOf('@')) {
       case -1: n = `_@${name}`; break
       case 0: n = `_${name}`; break
       default: n = name
     }
     const qName = await queryName(n)
     console.log(`${name}->${n}->${qName}`)
-    return qName == pubkey
+    return qName === pubkey
   }
-  
-  public async updateProfileFromMeta(pubkeys) {
+  public async updateProfileFromMeta(pubkeys: string[]) {
```

Comment:
empty line before function, please

Reply:
Made the adjustment in the latest commit